pandas/tests/test_base.py

```diff
@@ -296,14 +296,16 @@ def test_none_comparison(self):
                 # result = None != o  # noqa
                 # assert result.iat[0]
                 # assert result.iat[1]
-
-                result = None > o
-                assert not result.iat[0]
-                assert not result.iat[1]
-
-                result = o < None
-                assert not result.iat[0]
-                assert not result.iat[1]
+                if not (is_datetime64_dtype(o) or is_datetimetz(o)):
+                    # Following DatetimeIndex (and Timestamp) convention,
```

Comment:
I assume inequality comparison raising is tested elsewhere? Or should we assert that in an `else`?

Reply:
Forgot to port that test from the previous PR, good catch.